contents/scalability/scalable-development.md

```diff
@@ -33,9 +33,29 @@ There are a few common bottlenecks that can be avoided such as database querying
 
 There are many program profilers available that you can use to find the bottlenecks in your server. By finding out where the server spends most computational time in, you can potentially save huge amounts of time.
 
-Performance bottlenecks ([resource](https://www.apicasystem.com/blog/5-common-performance-bottlenecks/))
+Performance bottlenecks ([resource](https://www.apicasystem.com/blog/5-common-performance-bottlenecks))
 
-Non-Relational Database ([resource](http://www.jamesserra.com/archive/2015/08/relational-databases-vs-non-relational-databases/)) ([resource](https://www.pluralsight.com/blog/software-development/relational-non-relational-databases))
```

Comment:
I think it would be more readable if you number the points.

1. Performance bottlenecks
1. Non-relational database

Furthermore, consider describing the resources you link in the page so that readers can have an idea of what the link is about without needing to break the flow of reading the rest of the webpage